bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -23,7 +23,8 @@
     CONFIGURATION_ERROR(7),
     BRIDGE_OFFLINE(8),
     FIRMWARE_UPDATING(9),
-    DUTY_CYCLE(10);
+    FIRMWARE_UPDATE_PENDING(10),
```

Comment:
I would change the documentation as soon as this PR is merged if this is ok for you